From 9312cda13a18c4b790804540095e4f1661a5cd0f Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 16 May 2020 15:31:05 +0200
Subject: [PATCH 18/N] fixup! Do not create cygwin symlinks. Instead use deep
 copy of files/folders.

Whoops. I forgot this in my PR.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/path.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/winsup/cygwin/path.cc b/winsup/cygwin/path.cc
index 573a9b7..927efd8 100644
--- a/winsup/cygwin/path.cc
+++ b/winsup/cygwin/path.cc
@@ -2281,6 +2281,7 @@ symlink_worker (const char *oldpath, path_conv &win32_newpath, bool isdevice)
                  to the directory of the destination */
              {
                 /* Determine the character position of the last path component */
+                const char *newpath = win32_newpath.get_posix();
                 int pos = strlen (newpath);
                 while (--pos >= 0)
                   if (isdirsep (newpath[pos]))
-- 
2.27.0

